build-tools/rpm/macros.yast

```diff
@@ -15,49 +15,89 @@
 %yast_controldir %{yast_dir}/control
 %yast_schemadir %{yast_dir}/schema
 %yast_scrconfdir %{yast_dir}/scrconf
-%yast_desktopdir %{_prefix}/share/applications/YaST2
+%yast_desktopdir %{_datadir}/applications/YaST2
 
 %yast_ybindir %{_prefix}/lib/YaST2/bin
 %yast_ystartupdir %{_prefix}/lib/YaST2
-%yast_plugindir %{_libdir}/YaST2/plugin
-%yast_includedir %{_prefix}/include/YaST2
-%yast_docdir %{_prefix}/share/doc/packages/%{name}
-%yast_logdir /var/log/YaST2
-%yast_vardir /var/lib/YaST2
-%yast_fillupdir /var/adm/fillup-templates
+%yast_libdir %{_libdir}/YaST2
```

Comment:
@besser82 I have to revert it as it is already defined on https://github.com/yast/yast-devtools/pull/88/files#diff-0fc23eb95cef73cdbc8baba0bdc8729aR14 . This way it breaks building of yast packages, os we really want to use real libdir to have `/usr/lib64/YaST2` and yast_libdir should always point to `/usr/lib/YaST2`